sys/net/sock/sock_util.c

```diff
@@ -36,19 +36,18 @@
 #define PORT_STR_LEN    (5)
 #define NETIF_STR_LEN   (5)
 
-int sock_udp_ep_fmt(const sock_udp_ep_t *endpoint, char *addr_str, uint16_t *port)
+int sock_tl_ep_fmt(const struct _sock_tl_ep *endpoint,
+                   char *addr_str, uint16_t *port)
 {
     void *addr_ptr;
     *addr_str = '\0';
 
     switch (endpoint->family) {
-#if defined(SOCK_HAS_IPV4)
```

Comment:
I explained that removal in the commit message: There is no `SOCK_HAS_IPV4` define. All `sock`s support IPv4 as a basis (at least at compile time, if they can't actually service IPv4 they return an error), only IPv6 is the special case (due to their long addresses in the endpoint definition).

Reply:
Thanks for pointing that out!